docs/pages/guides/using-graphql.md

```diff
@@ -157,8 +157,6 @@ There are four major GraphQL clients available at the moment:
 
 Apollo, Relay and Urql implement further features like caching, realtime support with GraphQL subscriptions or optimistic UI updates.
 
-Learn how to integrate with Auth0 social providers in the [expo-auth0-example](https://github.com/graphcool-examples/react-native-graphql/tree/master/authentication-with-expo-and-auth0) repository.
```

Comment:
This is now outdated and not supported, graphcool also doesn't exist anymore (now prisma). 😅 This page could use a refresh